letters/modernized/15-letters.adoc

```diff
@@ -371,14 +386,19 @@ Calmly therefore, warm at heart, with brotherly love,
 with hopes full of immortality for you, and for myself,
 I dearly salute you and your spouse, remaining Your affectionate friend,
```

Comment:
My eye just happened to catch that the "Your" here was capitalized... which probably means that in the original letter it was set up as a SSCL.  When I see things like this, I usually 1) put a comma after the last word in the paragraph (in this case the word "remaining,"  2) then bump "Your affectionate friend," to its own line, and mark it SSCL. 👍🏻